### PR TITLE
added kick reaction

### LINKED ISSUE #69: Implement Kick reaction class
Currently, there is no implementation for the Kick reaction in the Splendor mobile game WebSocket server. This reaction is important for the host of the game to be able to kick other players from the lobby. This will send a message to kicked player and remove him from the game, and also an announcement to all players in the game that the other player.

More info in the java doc of this class.
https://github.com/Splendor-Mobile-Game/ServerSide/blob/7bf308007d5896d58199e58e5a6157ae6235b744/src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/Kick.java#L9-L54

### PR DESCRIPTION
**Pull request by VEXI19**
I don't know if that was needed but I also created a niew exception when a user has no permission for an action (PerrmissionDeniedExeption)

### CODE REVIEW

File: src/main/java/com/github/splendor_mobile_game/websocket/handlers/exceptions/PerrmissionDeniedExeption.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package com.github.splendor_mobile_game.websocket.handlers.exceptions;
+
+public class PerrmissionDeniedExeption extends Exception {
```
Comment (kestivvi) [inline src/main/java/com/github/splendor_mobile_game/websocket/handlers/exceptions/PerrmissionDeniedExeption.java:6]: Add one line java doc.

File: src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/Kick.java [hunk 2]
```diff
@@ -1,67 +1,145 @@
...
- *      "type": "KICK_RESPONSE",
- *      "result": "FAILURE"
- *      "data": {
- *          "error": "You cannot kick the player in the game you're not the host of!"
- *      }
- * }
- * 
- */
 @ReactionName("KICK")
 public class Kick extends Reaction {
```
Comment (kestivvi) [inline src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/Kick.java:53]: Add comprehensive java doc for this class. See how `StartGame` reaction does this or `GetTokens` (that will be merged) for example.

**Review by kestivvi — CHANGES_REQUESTED** [review]
Good job man. Just add some java docs.

**Review by Vertonowsky — DISMISSED** [review]

**Review by kestivvi — APPROVED** [review]
Great

**Review by agatamrnw — APPROVED** [review]

**Pull request merged**